GTViewController/Classes/Parsing Classes/GTPageInterpreter.m

```diff
@@ -77,12 +69,17 @@
 NSString * const kAttr_y				= @"y";
 NSString * const kAttr_width			= @"w";
 NSString * const kAttr_height			= @"h";
+
+// Constants for XML attributes for follow up modal, events and listeners added in v7.0.x of this pod
 NSString * const kAttr_tap_events		= @"tap-events";
+NSString * const kAttr_followup_id      = @"followup-id";
+NSString * const kAttr_context_id       = @"context-id";
 extern NSString * const kAttr_listeners;
+NSString * const kAttr_type             = @"type";
 
 NSString * const kAttr_yoff				= @"yoffset";
 NSString * const kAttr_xoff				= @"xoffset";
-
+NSString * const kAttr_xTrailingOff     = @"x-trailing-offset";
```

Comment:
The y cases are rare but there are a couple. There is a purple page toward the back of the kgp that needed yoffsets. I'm happy for it to go in JIRA and tackle it another day.